components/sidebar/add_channel_dropdown.tsx

```diff
@@ -12,6 +12,35 @@ import OverlayTrigger from 'components/overlay_trigger';
 import Tooltip from 'components/tooltip';
 import {CreateAndJoinChannelsTour, InvitePeopleTour} from 'components/onboarding_tour';
 
+function useRoveFocus(size: number, isCompUnmounted: boolean, setIsCompUnmounted: (e: boolean) => void) {
```

Comment:
Should we handle this in the `menu_items` component so that we can use this for all menus?

Reply:
@AbhinavVihan Yes but you can translate the code into the class component. Add the event listener to `componentDidMount` and call your `handleKeyDown` function. You can find examples in the webapp of where we do this in class components

Reply:
@BenCookie95, Would you please take a look.